File: Two Sum II - Input Array Is Sorted/Two Sum II - Input Array Is Sorted.py
class Solution(object):
 def twoSum(self,numbers, target):
  final=[]
  low,high=0,len(numbers)-1
  while low < high:

   nums=numbers[low] + numbers[high]
   if nums == target:
    return [low+1,high+1]
   elif nums>target:
    high=high-1
   else:
    low=low+1

File: Two Sum II - Input Array Is Sorted/Two Sum II - Input Array Is Sorted.java
class Solution {
    public int[] twoSum(int[] numbers, int target) {
        int f = 0, s = numbers.length - 1;
        int[] ans = new int[2];
        
        while (f < s) {
            int sum = numbers[f] + numbers[s];
            if (sum < target) {
                f++;
            }
            else if (sum > target) {
                s--;
            }
            else {
                ans[0] = f + 1;
                ans[1] = s + 1;
                break;
            }
        }
        return ans;
    }
}